encoders/nlp/LaserEncoder/aux.py

```diff
@@ -0,0 +1,22 @@
+import pkg_resources
+
+def test(requirements_path):
```

Comment:
I'm not sure I understand how this would be used. Is it part of CI, or is it up to the dev to run it?

Reply:
Why are we setting that version? Can't we just do it in the requirements?

Reply:
oh this is a file that should not have been uploaded will remove